website/src/pages/swiss-wastewater/flu.astro

```diff
@@ -28,32 +33,35 @@ const references = (await (await fetch(getReferencesUrl)).json()).data.map(
     dataOrigins={[dataOrigins.wise]}
     lapisUrl={lapisUrl}
 >
-    <gs-app lapis={lapisUrl}>
-        <div class='m-4 flex flex-col gap-4'>
-            {
-                references.map((reference) => (
-                    <>
-                        <h2 class='text-2xl'>{reference}</h2>
-                        {(['nucleotide', 'amino acid'] satisfies SequenceType[]).map((sequenceType) => (
-                            <GsWastewaterMutationsOverTime
-                                title={`${sequenceType} mutations`}
-                                lapisFilter={{
-                                    reference,
-                                    versionStatus: 'LATEST_VERSION',
-                                    isRevocation: false,
-                                }}
-                                sequenceType={sequenceType}
-                            >
-                                <InfluenzaWastewaterInfo
-                                    slot='infoText'
-                                    reference={reference}
+    {
+        references.map((reference) => {
+            const mutationAnnotations = reference === 'N2' ? mutationAnnotationsN2 : mutationAnnotationsN1;
```

Comment:
Is that logic here correct? If the reference is not "N2", then use the annotations of "N1"? Especially since we don't have control over which `references` we have (because we fetch that from LAPIS)?

Wouldn't it be safer to put that logic into a function in `wastewaterConfig.ts` that does a 
```typescript
switch (reference) {
  case ...:
    return ...;
  default:
    return [];
}
```

Reply:
ah yes this is a better approach - I will change, thansk for spotting!